OrbitGl/CallStackDataView.cpp

```diff
@@ -191,10 +190,6 @@ CallStackDataView::CallStackDataViewFrame CallStackDataView::GetFrameFromRow(int
 
 CallStackDataView::CallStackDataViewFrame CallStackDataView::GetFrameFromIndex(
     int index_in_callstack) {
-  if (index_in_callstack >= static_cast<int>(callstack_.GetFramesCount())) {
```

Comment:
I guess you are removing this, as this does never happen. Could you then add a CHECK for now?

Reply:
done